### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by akpathan2799**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

**Comment by akpathan2799** [comment]
how much time will it take to merge

File: domains/adeebkhan.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Describe the use of this subdomain",
+    "repo": "https://github.com/<akpathan2799>/<akpathan2799>.github.io",
```
Comment (CuteDog5695) [inline domains/adeebkhan.json:3]: remove the <>

File: domains/adeebkhan.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Describe the use of this subdomain",
+    "repo": "https://github.com/<akpathan2799>/<akpathan2799>.github.io",
+    "owner": {
+        "username": "<akpathan2799>",
```
Comment (CuteDog5695) [inline domains/adeebkhan.json:5]: remove the <> in the username

File: domains/adeebkhan.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Describe the use of this subdomain",
+    "repo": "https://github.com/<akpathan2799>/<akpathan2799>.github.io",
+    "owner": {
+        "username": "<akpathan2799>",
+        "email": "akpathan2799@gmail.com",
+       
+    },
+    "record": {
+        "CNAME": "<akpathan2799>.github.io"
```
Comment (CuteDog5695) [inline domains/adeebkhan.json:10]: remove the <> in the CNAME

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/adeebkhan.json (hunk 3 above)
Comment (akpathan2799) [reply domains/adeebkhan.json:10]: ok wait a second

**Comment by akpathan2799** [comment]
i have done a new pull request please check

**Comment by akpathan2799** [comment]
can you check it

**Comment by akpathan2799** [comment]
Please can you reopen file upload for me username akpathan2799

**Pull request closed without merging**